chanzuckerberg/single-cell-data-portal
# feat: updates to marker genes panel

## Reason for Change

https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-data-portal/6132

## Changes

implements the changes detailed in the ticket.

in gene expression sidebar:
![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/5653616/3d9df526-5658-493f-b114-f9365e700555)

![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/5653616/ebdf5020-bc3b-49bf-bd66-0ed3afcf0688)

![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/5653616/108b56cc-da8a-4ea6-ad15-b03ee9c74f93)

in cellguide:

![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/5653616/0818f249-097c-43ad-a0b0-b17595fd1776)

![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/5653616/01594574-084e-43bd-ac21-30c458dd312d)

![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/5653616/770155bc-25c1-4405-82ba-8f5e43616b73)

![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/5653616/5a375124-510e-431b-bc2b-94bd545f6ef3)

![image](https://github.com/chanzuckerberg/single-cell-data-portal/assets/5653616/45633d61-d9df-4090-9606-f6388e5bd7ce)

note that this PR also moves some stuff into common directories so we can use things for both the marker genes table in cellguide and in GE

## Testing steps

1. go to [rdev](https://pr-6260-frontend.rdev.single-cell.czi.technology/)
2. open up any the sidebar for any cell in gene expression, and check the tooltip copy + column names
3. click into cell guide, and check the tooltip copy + column names

## Checklist 🛎️


## Notes for Reviewer
